src/logion/migration/1687351664609-AddLegalCertificateFees.ts

```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class AddLegalCertificateFees1687351664609 implements MigrationInterface {
```

Comment:
`AddCertificate...` would be enough.

Reply:
Is `AddLegalAndCertificateFees` better for you ?

Reply:
Sorry, my bad. I forgot about the legal fees being added as well. Nevermind.